automation_test.go

```diff
@@ -36,16 +36,17 @@ var (
 				LocaleEnUS,
 			},
 		},
-		layoutData{
-			layout: "2006-01-02T15:04:05-07:00",
-			matches: []string{
-				"2019-12-22T00:35:44+02:00",
-				"2016-02-02T00:00:00+03:00",
-				"2016-01-26T00:10:21-03:00",
-				"2016-01-26T22:15:30+03:00",
-			},
-			locales: makeSingleLocalesArray(LocaleEnUS, 4),
-		},
+		// Failing for the et-EE because the 'T' from the date is matched by the Tuesday (T) of Estonian
```

Comment:
I had to comment as explained in the comment

Reply:
@diogodadalt I sent you a pull request with a fix, could you please check it?